experimental/examples/attributable-map-demo/src/container.ts

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ModelContainerRuntimeFactory } from "@fluid-example/example-utils";
+import { IContainer } from "@fluidframework/container-definitions";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+
+import { AttributableMapPage } from "./fluid-object";
+
+export interface IAttributableMapPageAppModel {
+	readonly attributableMapPage: AttributableMapPage;
+}
+
+class AttributableMapPageAppModel implements IAttributableMapPageAppModel {
+	public constructor(public readonly attributableMapPage: AttributableMapPage) {}
+}
+
+const attributableMapPageId = "attributable-map-page";
+
+export class AttributableMapPageContainerRuntimeFactory extends ModelContainerRuntimeFactory<IAttributableMapPageAppModel> {
```

Comment:
since this demo should show off the whole expected usage of attribution, you should leverage `mixinAttributor` from `@fluid-experimental/attributor` and use its functionality to render a depiction of the who last edited the content and when they did so, rather than just the AttributionKey.